platform/engineering/design-docs/2022-03-11-schedule-pending-appt.md

```diff
@@ -0,0 +1,156 @@
+# Scheduling Pending Appointments
+
+**Author(s):** Andrew Herzberg  
+**Last Updated:** March 11,2022  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** _Person A_ \[ \], _Person B_ \[ \], ...  
+
+## Overview
+
+### Objective
+The goal is to add the feature to the mobile app to schedule pending appointments. This functionality is currently available in `vets-website` so much of the logic can be utilized in the mobile app.
+
+The intended audience is for the engineering team.
+
+### Background
+Currently, the `vets-website` scheduling feature is as follows:  
```

Comment:
Maybe a [sequence diagram](https://plantuml.com/sequence-diagram) could help illustrate the current state?

Reply:
No problem. Created. Were you thinking the diagram and the current background or in place of?